src/main/java/edu/ucla/library/prl/harvester/Institution.java

```diff
@@ -186,19 +186,19 @@ public Institution(final JsonObject aJsonObject) {
         myID = Optional.ofNullable(aJsonObject.getInteger(ID));
 
         if (name != null) {
-            myName = name;
+            myName = name.trim();
```

Comment:
Maybe also do `Objects.requireNonNull(StringUtils.trimToNull(name))` here too?

Reply:
But the code is already testing for a null value (line 188); and can we throw the specific exception object/message (line 191) if we use `Objects#requireNonNull`?

Reply:
yes, but consider the case where `name` is something like `"   "` (a string with only whitespace); `"   ".trim().equals("")`, right? The code snippet I suggested would make such a state un-representable, which seems desirable to me; I don't think we should allow whitespace-only strings to persist.